smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftSymbolProvider.kt

```diff
@@ -284,6 +295,109 @@ class SwiftSymbolProvider(private val model: Model, val swiftSettings: SwiftSett
         return ModelFileUtils.filename(swiftSettings, name)
     }
 
```

Comment:
The `handleDefaultValue` below extracts default value for the given shape, saves it to properties of the given symbol builder, then returns the modified builder back to the caller. If the shape does not or should not have a default value, it returns unmodified builder.